src/scripts/routes.js

```diff
@@ -234,8 +234,9 @@ define([
         transition: 'fade'
     });
     defineRoute({
-        path: '/itemdetails.html',
-        controller: 'itemDetails',
+        alias: '/details',
+        path: '/controllers/itemDetails/itemDetails.html',
+        controller: 'itemDetails/itemDetails',
```

Comment:
Ah, I see you added that to this pull request. I think the path should assume `controllers` at the beginning since we do the same for the controller. It might also be useful at this point to assume `index.html` and `index.js` and just assume the end of the path is a folder containing both files. This would be relatively standard for pages with Vue.

Reply:
The issue is that, until we move every HTML page, assuming the beginning of the path isn't going to work.

I'd also argue against assuming things since it makes it slightly harder to understand. Here, you know at a glance the path of the thing.

Getting rid of module aliases while also introducing this kind of thing seems a bit backward :)